test/functional/server/server_spec.lua

```diff
@@ -17,34 +18,32 @@ describe('serverstart(), serverstop()', function()
     eq('', eval('$NVIM_LISTEN_ADDRESS'))
   end)
 
-  it([[sets v:servername _only_ on nvim startup
-      (unless all servers are stopped)]], function()
-    local initial_server = eval('v:servername')
+  it('sets v:servername _only_ on nvim startup unless all servers are stopped',
+  function()
+    local initial_server = meths.get_vvar('servername')
```

Comment:
This is not preferable IMO. Vimscript is more recognizable (for most Vim devs hopefully).